<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LIN-4PX71F3" failures="1" tests="2" name="ErrorvalidationTest Test" time="15.547" errors="0" timestamp="2024-02-15T16:57:47 IST">
  <testcase classname="FrameworkDesigne2e.Test.ErrorvalidationTest" name="LoginErrorValidation" time="0.416"/>
  <testcase classname="FrameworkDesigne2e.Test.ErrorvalidationTest" name="ProductErrorValidation" time="10.61">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:86)
at org.junit.Assert.assertTrue(Assert.java:41)
at org.junit.Assert.assertTrue(Assert.java:52)
at FrameworkDesigne2e.Test.ErrorvalidationTest.ProductErrorValidation(ErrorvalidationTest.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- ProductErrorValidation -->
</testsuite> <!-- ErrorvalidationTest Test -->
